DIPs/DIP-36.md

```diff
@@ -0,0 +1,53 @@
+
+## Proposal
+
+As with previous productions, there are some high-level goals that guide the narrative/design of the game: 
+
+* **Educational**. Players of all experiences should be able to learn something they didn't yet know about the world. 
+
+* **Exploratory**. The game should generously reward curiosity, and challenge players to pay attention to signals disguised as noise. As a secondary goal, it should encourage players to explore the venue and its features/programming/services, as well as incorporate some quests to touch grass. 
+
+* **Physical**. While some elements of the game must be software, the game is played live in the time and place of the event. Physical artifacts make the story tick.  
+
+For devcon7, we propose a core narrative puzzle structure similar to previous games, whereby players advance through chapters of increasing difficulty to reveal more of the story. In every Daedalus production, significant chapters include special *keys* belonging to NPCs. These keys define the narrative arc of the game, and collecting all of them to complete the story is the central goal of every player. A leaderboard or wallet tool can help players keep track of their progress by chapter and number of keys collected, and any player that concludes all chapters with all keys is a "winner". 
+
+As the total number of keys in the game is arbitrary, the entry point for collaboration would be in *side-quest keys* that would, hypothetically, need to follow a certain format to be 'in-world' (and trackable by the leaderboard, for example). 
+
+// 2024-02-23 Reserving this space as a commitment to update this DIP on the side questing objective after discussion on devcon forum and with the Daedalus/SDP team. 
+
+
+## Requirements
+
+From Devcon and the devcon team, we need: 
+
+* NPC volunteers. We will need to employ a handful of people to act as non-player characters in the game -- we would like to hook into the devcon volunteer pipeline for NPC roles within the game, and work with the volunteer coordinator on how best to organize this aspect. 
+* Access and trust for players of the game to 'unlock' special places in the venue. Just as in previous years, we will mark off these special 'in-game' doors, objects, and non-player characters with an established symbol of trust. 
+* A list or description of all other planned activities and points of contact for places of interest within the venue. Help us help you highlight the best aspects of the devcon experience, like the chill spot, meeting rooms, and any other programming. 
+* The core team needs elevated access permissions to the devcon teams operations (discord) so that we can set up features of the game, and more importantly so that we can prepare and prime venue staff/security about these specific locations and expected (weird) behaviors that might be mis-interpreted to be threatening without prior warning.  
```

Comment:
```suggestion
* The core team needs elevated access permissions to the devcon teams operations (discord) so that we can set up features of the game, and more importantly so that we can prepare and prime venue staff/security about these specific locations and expected (weird) behaviors that might be misinterpreted to be threatening without prior warning.  
```